### ISSUE TITLE
False positive: eslint overrides

### ISSUE THREAD

**Issue by clar-cmp** [depcheck/depcheck#570]
In addition to including plugins in eslintConfig directly, you can also specifically require them in overrides. For example, in the `overrides` for tests, we include `eslint-plugin-mocha`.

This should be as easy as running the same checks for plugins on the elements of the `overrides` array as well as in the root config.

**Comment by rumpl** [comment]
@clar-cmp thanks for openning the issue, it would be awesome if we had a repository that we can take a look at please

**Comment by clar-cmp** [comment]
@rumpl here's the relevant piece of the eslintConfig from a project:

```json
    "overrides": [
      {
        "files": [
          "test/**/*.js",
          "test/**/*.ts"
        ],
        "extends": [
          "plugin:chai-expect/recommended",
          "plugin:chai-friendly/recommended",
          "plugin:mocha/recommended"
        ],
        "rules": {
          "@typescript-eslint/naming-convention": "off",
          "@typescript-eslint/no-unsafe-call": "off",
          "@typescript-eslint/no-unsafe-member-access": "off",
          "func-names": "off"
        }
      }
    ]
```

So, in this case, the override includes `plugin:mocha/recommended` which uses `eslint-plugin-mocha`, as a simple example.

Basically, `overrides` follow the same structure as the regular eslint config, but apply to specific files, like this case which applies to the `test/` dir.

**Comment by clar-cmp** [comment]
Still an issue, just checked.

**Comment by Xyaren** [comment]
I too experience that issue.

**Comment by Xyaren** [comment]
So, I made a PR #641 with a fix.
Maybe @rumpl  can take a look :)

**Comment by rumpl** [comment]
Will take a look, awesome, thanks for the PR!
